docs/book/src/developer/providers/v1.3-to-v1.4.md

```diff
@@ -70,7 +70,8 @@ maintainers of providers and consumers of our Go API.
   For more information, please see: https://github.com/kubernetes/enhancements/issues/2845
 - A new `KCPRemediationSpec` test has been added providing better test coverage for KCP remediation most common use cases. As a consequence `MachineRemediationSpec` now only tests remediation of
   worker machines (NOTE: we plan to improve this test as well in a future iteration).
-
+- Package `test/infrastructure/docker/internal/third_party/forked/loadbalancer` has been moved to `test/infrastructure/docker/internal/loadbalancer` to allow it to diverge from the upstream Kind package.
```

Comment:
nit/Q: Is this relevant to provider authors?

Reply:
As it's still internal maybe not - but because it's a slightly strange sort of change - third_party -> first_party - I wanted to highlight it somewhere.

Reply:
No objection